### DISCUSSION TITLE
How to prevent a Steam game from losing focus?

### DISCUSSION THREAD

**Discussion by antoyo** [i3/i3#5599] [Q&A]
Hi.
Is there a way to prevent an app from losing focus?
I have a multi-monitor setup and I play games on Steam.
It can happen that some apps (reminders, chat app) pop up and steal focus, pausing the game.
How can I prevent that?
Thanks.

**Comment by orowith2os** [comment]
Gamescope might be worth a try - it makes games think they're always in focus, iirc. Solves quite a few issues related to focus.

**Comment by orestisfl** [comment]
i3 has a `no_focus` directive: https://i3wm.org/docs/userguide.html#no_focus
